core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -741,7 +741,8 @@ class ReplicaManager(val config: KafkaConfig,
 
           // throw NotLeaderOrFollowerException if replica does not exist for the given partition
           val partition = getPartitionOrException(topicPartition)
-          partition.localLogOrException
```

Comment:
This is weird... Do you know why it was here?

Reply:
The code surrounding is from 14 months - 2 years ago, so not sure.